scripts/main/lychee.js

```diff
@@ -50,7 +50,7 @@ let lychee = {
 
 	album_subtitle_type: "oldstyle",
 
-	upload_processing_limit: 4,
+	upload_processing_limit: 2,
```

Comment:
Why did you change it to `2`? And why only in this least important place?

Reply:
I was testing. :'D